### PR TITLE
Add very basic gallery

### PR DESCRIPTION
**Pull request by phlmn**

### CODE REVIEW

File: backend/transcribee_backend/api/views.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 class DocumentViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
-        return Document.objects.filter(user=self.request.user)
+        return Document.objects.filter(user=self.request.user).order_by("-created_at")
```
Comment (pajowu) [inline backend/transcribee_backend/api/views.py:26]: I think `-changed_at` would be the better one here, so the user always sees the document it just edited first.

Comment (pajowu) [reply backend/transcribee_backend/api/views.py:26]: Also I think we might want to add an ordering to the document model, so we always get that ordering? `created_at` might be the right one there, so we have a more permanent sorting https://docs.djangoproject.com/en/4.1/ref/models/options/#ordering

in base/models.py:
```
class Document(models.Model):
    class Meta:
        ordering = ["-created_at", "id"]
```

File: frontend/index.html [hunk 2]
```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>transcribee 🐝</title>
-    <link rel="stylesheet" href="src/index.css">
-    <link rel="stylesheet" href="node_modules/@fontsource/inter/latin.css">
+    <link rel="stylesheet" href="/src/index.css">
+    <link rel="stylesheet" href="/node_modules/@fontsource/inter/latin.css">
```
Comment (pajowu) [inline frontend/index.html:8]: Can we make this (and the routes) depend on a base url setting? So we can host transcribee somewhere else than the document root?

File: frontend/src/editor/TranscriptionEditor.tsx [hunk 3]
```diff
@@ -47,15 +47,11 @@ export default function TranscriptionEditor() {
     console.log('new yDoc');
```
Comment (pajowu) [inline frontend/src/editor/TranscriptionEditor.tsx:47]: 🖨️

File: frontend/src/pages/DocumentPage.tsx [hunk 4]
```diff
@@ -1,10 +1,11 @@
+import { RouteComponentProps } from 'wouter';
 import TranscriptionEditor from '../editor/TranscriptionEditor';
 
-export default function DocumentPage() {
+export default function DocumentPage({ params }: RouteComponentProps<{ documentId: string }>) {
   return (
-    <div className="container p-8 mx-auto">
+    <div className="container p-6 mx-auto">
       <h2 className="text-2xl font-bold mb-8">Test Editor</h2>
```
Comment (pajowu) [inline frontend/src/pages/DocumentPage.tsx:7]: Wish (not necessarily for this PR): Show the actual document title here

File: frontend/src/pages/DocumentPage.tsx [hunk 5]
```diff
@@ -1,10 +1,11 @@
+import { RouteComponentProps } from 'wouter';
 import TranscriptionEditor from '../editor/TranscriptionEditor';
 
-export default function DocumentPage() {
+export default function DocumentPage({ params }: RouteComponentProps<{ documentId: string }>) {
```
Comment (pajowu) [inline frontend/src/pages/DocumentPage.tsx:4]: Now that we have the gallery which serves as a "home page" I would wish for a link from the document page back to the gallery

File: frontend/src/pages/HomePage.tsx [hunk 6]
```diff
@@ -1,12 +1,78 @@
...
-      <PrimaryButton onClick={() => navigate('/new')}>New Document</PrimaryButton>
+    <div className="container mx-auto p-6">
+      <div className="mb-8 flex justify-between items-center">
+        <h1 className="text-xl font-bold">transcribee</h1>
+        <div>
+          <PrimaryButton onClick={() => navigate('/login')}>Login</PrimaryButton>{' '}
+          <PrimaryButton onClick={() => navigate('/new')}>New Document</PrimaryButton>
+        </div>
+      </div>
+      <ul
```
Comment (pajowu) [inline frontend/src/pages/HomePage.tsx:40]: Not necessarily in this PR, maybe only once they get more complex: Can we split the list & list items out to separate components?

**Review by pajowu — COMMENTED** [review]
Looks great in general, I added a few smaller change requests & wishes as inline comments.

File: frontend/src/pages/HomePage.tsx [hunk 7]
```diff
@@ -37,7 +37,17 @@ export default function HomePage() {
...
+        className={clsx(
+          'grid',
+          'grid-cols-2',
+          'sm:grid-cols-3',
+          'md:grid-cols-4',
+          'lg:grid-cols-5',
+          'xl:grid-cols-6',
+          'gap-6',
+        )}
+      >
```
Comment (pajowu) [inline frontend/src/pages/HomePage.tsx:50]: What's the advantage of using clsx as opposed to a plain string here?

File: frontend/index.html (hunk 2 above)
Comment (phlmn) [reply frontend/index.html:8]: Done, the router now respects vite's base path (https://vitejs.dev/guide/build.html#public-base-path).

File: frontend/src/pages/HomePage.tsx (hunk 6 above)
Comment (phlmn) [reply frontend/src/pages/HomePage.tsx:40]: Yes, we should totally do that, when the item get's more details 👍

File: frontend/src/pages/HomePage.tsx (hunk 7 above)
Comment (phlmn) [reply frontend/src/pages/HomePage.tsx:50]: This simply reduces the line length

**Comment by phlmn** [comment]
@pajowu I would address the editor related settings when revising the editor in general. For now I would work towards a holistic protoype.

File: backend/transcribee_backend/sync/consumers.py [hunk 8]
```diff
@@ -52,6 +57,7 @@ async def disconnect(self, close_code):
 
     async def receive(self, bytes_data):
         await create_update(document=self.document, update_content=bytes_data)
+        await save_doc(self.document)  # save document to update changed_at field
```
Comment (pajowu) [inline backend/transcribee_backend/sync/consumers.py:60]:
```suggestion
        await database_sync_to_async(self.document.save)()  # save document to update changed_at field
```

**Review by pajowu — CHANGES_REQUESTED** [review]

**Review by pajowu — APPROVED** [review]

**Pull request merged**